timesketch/frontend-ng/src/components/LeftPanel/TimelinesTable.vue

```diff
@@ -15,14 +15,19 @@ limitations under the License.
 -->
 
 <template>
-  <div class="content">
+  <div v-if="iconOnly" class="pa-4" style="cursor: pointer" @click="$emit('toggleDrawer')">
+    <v-icon left>mdi-clock-outline</v-icon>
+    <div style="height: 1px"></div>
+  </div>
+
+  <div v-else>
     <div
       class="pa-4"
-      :style="'cursor: pointer'"
+      style="cursor: pointer"
       @click="expanded = !expanded"
       :class="$vuetify.theme.dark ? 'dark-hover' : 'light-hover'"
     >
-      <span> <v-icon left>mdi-timeline-clock-outline</v-icon> Timelines </span>
+      <span> <v-icon left>mdi-clock-outline</v-icon> Timelines </span>
```

Comment:
Personally I prefer the `mdi-timeline-clock-outline` since it does not just represent the "time" factor but also the "line" or "table" factor. It also reflects the list view for timelines that we now have in the left panel.

What is your opinion on going with `mdi-timeline-clock-outline`?

Reply:
I don't like because it doesn't align with the rest of the icons very well and I think it clutters the UI. It feels out of place. I prefer something simpler.